ddp-workspace/projects/ddp-pancan/src/app/components/participation/participation.component.scss

```diff
@@ -0,0 +1,76 @@
+@import '../../../styles/common.scss';
+
+.participation {
+    text-align: left;
+
+    .title {
+        @extend .static-page-title;
```

Comment:
I can see that we use static-page-title class in html for step__title . Can we do it for this title element as well? (same for other extends)

Reply:
Fixed at 316abf2f0e57626829a3c6149f4cf9089db50904